### DISCUSSION TITLE
Turn on Settings Sync... on Codespace creation is it possible?

### DISCUSSION THREAD

**Discussion by souhailmerroun** [community/community#52247] [Codespaces] [answered]
### Select Topic Area

Question

### Body

<img width="401" alt="image" src="https://user-images.githubusercontent.com/8895242/230600425-04a782fd-4ed9-45f4-9ceb-879f7bc53d32.png">

**Comment by souhailmerroun** [comment]
I frequently use Github Codespaces for my projects, every time I build a new one, I have to turn on the toggle sync with my project. Is it possible to make on always?

**Comment by aishraj** [comment, marked as answer]
Hello there! We're sorry that you're running into this. Have you looked at our help [docs](https://docs.github.com/en/codespaces/customizing-your-codespace/personalizing-github-codespaces-for-your-account#settings-sync). If you're using your codespace in the browser, then this is an expected behavior.

**Comment by souhailmerroun** [comment]
okay perfect. thanks for the link.
